examples/graphql-ws/package.json

```diff
@@ -9,7 +9,7 @@
   "dependencies": {
     "@graphql-yoga/node": "2.13.11",
     "graphql": "16.6.0",
-    "graphql-ws": "0.10.11",
+    "graphql-ws": "5.10.1",
```

Comment:
this wasn't an existent latest version not sure how renovate worked https://www.npmjs.com/package/graphql-ws